packages/ui/src/proposals/types/proposals.ts

```diff
@@ -19,3 +19,13 @@ export interface Proposal {
   type: string
   proposer: Member
 }
+
+export interface PastProposal {
+  id: string
+  endedAt: string
+  title: string
+  rationale: string
+  stage: ProposalStage
+  type: string
+  proposer: Member
+}
```

Comment:
It looks like most of the fields are duplicated from a proposal. I think that we can safely assume that a proposals will have the same "type". THe past proposal will be a proposal that is in one of the stages that do not indicate that this is a current proposal (see [stages](https://github.com/Joystream/pioneer/pull/674/files)).

Let's have:

```suggestion
export interface PastProposal extends Proposal {
  endedAt: string
}
```

or just optional field on the `Proposal` interface.